scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -121,6 +121,14 @@ <h4 class="h4 d-inline-block">
 
       {{ matching_variants() }}
 
+      {% if audits | count_cursor > 0 %}
```

Comment:
Could this fit in the existing column over comments perhaps? Or in a new column next to the matching causatives, though perhaps the theme fit isn't perfect. While very nice, I feel the audit card will seldom be more than a third of a page wide, leaving a bit of unused screen on the right, while also being duplicated in the event log at the page bottom.

Reply:
I'll see what I can do!

Reply:
Like this it is ugly?

<img width="1428" alt="image" src="https://github.com/user-attachments/assets/2b35934a-c36c-4bf1-9625-4eefb5e2229b">

Reply:
I think it's perfectly fine. Overall, there are many cards now, and were before, but this one will only be there when it's informative.